src/mbgl/text/tagged_string.cpp

```diff
@@ -2,15 +2,49 @@
 #include <mbgl/math/minmax.hpp>
 #include <mbgl/util/i18n.hpp>
 
+namespace {
+char16_t PUAbegin = u'\uE000';
```

Comment:
why this magic number as delimiter?

Reply:
wouldn't a text ending with the delimiter break the code?

Reply:
My understanding is it is in a "private area" of the unicode table, but it is safe to use? i.e. some malicious label could have this codepoint on it.

Reply:
It is only used internally, cannot be set from outside and is not exposed as an external API